apps-rendering/src/components/embedWrapper.tsx

```diff
@@ -292,18 +300,17 @@ const divElementPropsToEmbedComponentProps = (
 						);
 					}
 					case EmbedKind.EmailSignup: {
-						return pipe(
-							elementProps,
-							parseGenericFields,
-							resultMap(
-								(
-									genericFields: GenericFields,
-								): EmailSignup => ({
-									kind: EmbedKind.EmailSignup,
-									...genericFields,
-								}),
-							),
-						);
+						return resultMap<string, EmailSignup>(
```

Comment:
@JamieB-gu we didn't pair on this, but I also filled in the implementation here in `embedWrapper.tsx`. Could you take a look here (and lines 89-99) please?